README.adoc

```diff
@@ -45,11 +45,13 @@ include:
 |No
 |Specifies that the server should bind to the given IP address and TCP port.  In general, servers can bind to more than one IP address and port, but this is not (yet?) supported. Defaults to "127.0.0.1:0".
 
-|`request_body_max_bytes`
+|`default_request_body_max_bytes`
 |`4096`
 |No
 |Specifies the maximum number of bytes allowed in a request body. Larger requests will receive a 400 error. Defaults to 1024.
 
+Can be overridden per-endpoint via the `request_body_max_bytes` parameter to `#[endpoint { ... }]`.
```

Comment:
future tense ;-)

(just joking; no need to change this)

Reply:
Haha, I'm actually going to move this to #1171 :)